Really/Program.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+
+namespace Really
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("You are dreaming.\nYou see yourself as a child");
+            Console.WriteLine("You are alone in a sandbox, digging a hole with your hands");
+            Console.WriteLine("You watch in amazement as the edges of the hole begin to crumble inwards");
+
+            Console.WriteLine("Do you allow curiosity to get the best of you? "); 
+          
+            string response = Console.ReadLine();
+
+
+            if(response == "yes")
```

Comment:
Consider putting a space between the "if" and the "(" to keep it consistent with the other parts of the code. `if (response == "yes")`